pkg/subnet/upgrades/upgrade_file.go

```diff
@@ -0,0 +1,49 @@
+package upgrades
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+)
+
+func WriteUpgradeFile(jsonBytes []byte, subnetName, upgradeFilesDir string) error {
```

Comment:
Is `upgradeFilesDir` ever not going to be the default dir from `app`? If not, just remove the argument and query the `app` dir in this method.

Reply:
`app` is not injected to this packge, so `upgradeFilesDir` is ok.